ci/scripts/build.sh

```diff
@@ -2,9 +2,7 @@
 
 cwd=$(pwd)
 
-export GOPATH=$cwd/go
-
-pushd $GOPATH/src/github.com/ONSdigital/dp-filter-api
+pushd $cwd/dp-filter-api
   make build && mv build/$(go env GOOS)-$(go env GOARCH)/* $cwd/build
```

Comment:
I dont think you need the GOOS and GOARCH environment variables anymore, see [dp-dataset API pr](https://github.com/ONSdigital/dp-dataset-api/pull/215)